### PR TITLE
milestone 2

### PR DESCRIPTION
**Pull request by mtan22**
sorry this is so late! I talked to Ben McMann about my situation, and I will get all the milestones done soon.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	<title>Michelle's Cookbook!</title>
+    <link rel="Stylesheet" href="index.css">
+  </head>
+  <body>
+      <a href="/Users/michelletan/bootcamp-project-2021/aboutme.html" target="_blank">About Me</a>
+      <p id="title">Michelle's Cookbook!</p>
+	  <p id="info">Hey! I'm Michelle and I love spending my free time cooking and experimenting with new recipes. Here you will find some of my favorite recipes of food I love to make!</p>
+      <br />
+      <h1>Recipes:</h1>
+      <a href="/Users/michelletan/bootcamp-project-2021/pasta.html" target="_blank">Gigi Hadid Pasta</a>
```
Comment (bbmcmann) [inline index.html]: So here you have a common error that most people ran into. in you < a > tags, you are using an absolute file path to link between your pages. This works fine on you computer, but does not work on any other machines, so your links are broken on my end.
To fix this you can just use the relative file path, which in this case would just be the name of the file (ex: pasta.html)
If you have questions I'm happy to help with this!

**Review by bbmcmann — COMMENTED** [review]
Hey Michelle! This code looks good for the most part, I noted one small bug that you can fix and then move on to the next milestone with. I really like your css styling and you have everything needed for milestone 1!

I was looking back through pull requests and don't see one for milestone 1 from you, and this code looks like milestone 1 stuff since there is no javascript. I just wanted to double check that this is a milestone 1 PR since this says milestone 2, if you did check off the first milestone and I'm just missing it let me know :)

File: index.html (hunk 1 above)
Comment (mtan22) [reply index.html]: Hi Ben, thank you so much for your input and your response! I fixed the file paths, let me know if it works! Also, I made a mistake last time committing my first milestone files only, but I just committed the second milestone files. Let me know if you can see them :)

**Comment by bbmcmann** [comment]
Hey Michelle, your code for milestone 2 looks good and all requirements are met. Also, the fixes you made to the links are working great now! You are all good to move onto the next milestone.

**Pull request closed without merging**